src/player_activity.cpp

```diff
@@ -104,8 +105,20 @@ cata::optional<std::string> player_activity::get_progress_message( const avatar
             type == activity_id( "ACT_CHOP_LOGS" ) ||
             type == activity_id( "ACT_CHOP_PLANKS" )
           ) {
+            const int percentage = ( ( moves_total - moves_left ) * 100 ) / moves_total;
```

Comment:
```suggestion
            const int percentage = ( moves_total - moves_left ) * 100 / moves_total;
```

Reply:
I just moved this line I didn't add any additional parens.
Also, I prefer the explicit parens to the implicit depending on the order of operations.